hplsql/src/main/java/org/apache/hive/hplsql/functions/FunctionMisc.java

```diff
@@ -59,6 +64,43 @@ public void register(BuiltinFunctions f) {
   void activityCount(HplsqlParser.Expr_spec_funcContext ctx) {
     evalInt(Long.valueOf(exec.getRowCount()));
   }
+
+  /**
+   * CAST function
+   */
+  void cast(HplsqlParser.Expr_spec_funcContext ctx) {
+    if (ctx.expr().size() != 1) {
+      evalNull();
+      return;
+    }
```

Comment:
Does it mean that the user specified wrong parameters? Should we throw exception?

Reply:
These are the APIs removed as a part of [HIVE-27492](https://issues.apache.org/jira/browse/HIVE-27492) fix. So just added back them again.Didn't do any code changes.

Reply:
As a part [HIVE-27492](https://issues.apache.org/jira/browse/HIVE-27492) fix, we fixed some issues like UDF using in select statement, that fix should be present so not reverting back entire [HIVE-27492](https://issues.apache.org/jira/browse/HIVE-27492) fix only the removed UDF cde is getting added and corrected for the functionality.